addons/website/static/tests/tours/website_no_dirty_page.js

```diff
@@ -24,6 +24,9 @@ tour.register('website_no_dirty_page', {
         content: "Confirm we are in edit mode",
         trigger: 'body.editor_has_snippets',
         run: () => null,
+    }, {
+        content: "Click on default paragraph",
+        trigger: '.s_text_image p.o_default_snippet_text',
```

Comment:
Kinda same question as If I make a test for each "+" menu config: I could also have a dedicated test for each of those instead of checking them all in the same tour... would help when one fails but might be overkill/slowing down test tours for no good reason, you tell me @rdeodoo

Reply:
If you do in one test: you will have to do a RPC to generate menu from within the test, but if you split in 3-4 tests and populate the menus from the python side before starting the test, no rpc will be needed (but it will be multiple tours).

So all in all I guess it's about the same perf wise? As long as all the tours are started in the same method back to back and not in multiple methods.

As for the other question: the test and fixes are already very nice, but if you feel like adding some more test let's go :bow:

Reply:
I would of course do it python side for the menus, not the question here :wink: 

- Conclusion of https://github.com/odoo/odoo/pull/144121#discussion_r1438261280: I will add 3 new tests (2 disabled as not fixed by this PR), for each "+" menu configuration, entering edit mode, discarding immediately.
- For the two other cases fixed by this PR (click on default text and click on a link), I could:
   - Do as I did: removing all menus to make sure the "+" menu is not annoying, clicking on default text, clicking on link, discard
   OR
  - Make 2 tests (one for each case), which both would remove the menus too to avoid the "+" menu, do the action (click on text / click on link), discard
  OR
  - Make 1 tests (which still removes the "+" menu), then start 2 tours (instead of one) (or rather one tour which leave/enter edit mode a lot), one per action to test to discard

Reply:
Fine as is imo

Reply:
Ok, I'll go with only one test with all menus removed for the text and links check :+1: We can split later if we start checking more cases anyway :)